### PR TITLE
Lots of UI fixes (whitespace and alignment)

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5745

*Description of changes:*
- overrode shadow style for header buttons
- added base styles for app colours
- fixed margins and padding everywhere
- changed top header to H5
- changed next headers to H6
- reformatted text that wrapped
- centered some stuff that wasn't
- deleted obsolete SASS overrides
- fixed Further Information page layout

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/App.vue [hunk 1]
```diff
@@ -247,6 +247,10 @@ export default class App extends Mixins(SessionTimerMixin) {
   background-color: rgb(33, 33, 33) // grey darken-4
   border-color: rgb(33, 33, 33) // grey darken-4
 
+.app-header
+  .v-btn
+    box-shadow: unset !important
```
Comment (severinbeauvais) [inline client/src/App.vue:252]: This is the simplest solution but I'm open to other ideas.

The problem is that the header uses different styles than the rest of the app. But here, our app styles were affecting the header buttons (eg, Login).

Or, we don't worry about it for now.

File: client/src/components/common/applicant-info-2.vue [hunk 2]
```diff
@@ -121,21 +118,24 @@
...
+        <v-col cols="2" class="h6">Additional Services</v-col>
+        <v-col cols="10" align-self="center">
+          <v-checkbox
+            hide-details
+            v-model="priorityRequest"
+            class="mt-0 pt-0"
+          >
             <template v-slot:label>
-              Priority Request - <b>$100 Fee</b>
+              <span>Priority Request - <b>$100 Fee</b></span>
```
Comment (severinbeauvais) [inline client/src/components/common/applicant-info-2.vue:131]: This was needed to preserve the space between the "-" and the "$100".

File: client/src/components/common/applicant-info-3.vue [hunk 3]
```diff
@@ -1,11 +1,9 @@
 <template>
-  <v-form v-model="isValid" ref="step2" id="applicant-info-3-form">
-    <v-container fluid class="pa-0">
+  <v-form v-model="isValid" ref="step3" id="applicant-info-3-form">
```
Comment (severinbeauvais) [inline client/src/components/common/applicant-info-3.vue:2]: This is actually an "alternate" step 2. (You get this page or applicant-info-2.)

I replaced all the places I could find this, which was just in this file actually.

File: client/src/components/new-request/main-container.vue [hunk 4]
```diff
@@ -1,10 +1,10 @@
 <template>
-  <v-container fluid class="white rounded copy-normal">
+  <v-container fluid class="white rounded copy-normal pa-10">
```
Comment (severinbeauvais) [inline client/src/components/new-request/main-container.vue:2]: This is the gutter (whitespace) around the main container (which is used for most or all pages). And all content should be tight up to this.

File: client/src/components/new-request/submit-request/reserve-submit.vue [hunk 5]
```diff
@@ -5,8 +5,7 @@
               :disabled="isContinue">
     <template v-slot:activator="scope">
       <v-btn @click="handleSubmit"
-             outlined
-             :class="{ 'reserve-submit-btn': !isContinue }"
+             :class="isContinue ? 'button-normal' : 'button-blue'"
```
Comment (severinbeauvais) [inline client/src/components/new-request/submit-request/reserve-submit.vue:8]: Global button styles ftw!

File: client/package-lock.json [hunk 6]
```diff
@@ -2277,9 +2277,9 @@
       }
     },
     "@types/yargs": {
-      "version": "15.0.10",
-      "resolved": "https://registry.npmjs.org/@types/yargs/-/yargs-15.0.10.tgz",
-      "integrity": "sha512-z8PNtlhrj7eJNLmrAivM7rjBESG6JwC5xP3RVk12i/8HVP7Xnx/sEmERnRImyEuUaJfO942X0qMOYsoupaJbZQ==",
+      "version": "15.0.11",
```
Comment (severinbeauvais) [inline client/package-lock.json:2280]: Note to Cam. This file seems pretty clean to me atm (after a full package reinstall), so no need to look into it further.

File: client/src/components/session-timer/countdown-timer-display.vue [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="countdown" :class="countdownClass">
```
Comment (severinbeauvais) [inline client/src/components/session-timer/countdown-timer-display.vue:2]: This was causing redundant classes (not a problem but...).

File: client/src/App.vue (hunk 1 above)
Comment (severinbeauvais) [reply client/src/App.vue:252]: @cameron-freshworks Haha, I see you did (almost) the same thing in your recent PR.

Comment (cameron-freshworks) [reply client/src/App.vue:252]: hahah i did! 
No, i just happened to notice it after rebasing :)

**Review by cameron-freshworks — APPROVED** [review]
Looks great!

**Comment by severinbeauvais** [comment]
@here It's unlikely I found everything to fix... Any further tweaks can be make in future tickets/PRs.

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**